src/main/java/core/basesyntax/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.ManufacturerDao;
+import core.basesyntax.db.Storage;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Manufacturer;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        Storage.addManufacturer(manufacturer);
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        for (Manufacturer manufacturer : Storage.manufacturers) {
+            if (manufacturer.getId() == id) {
```

Comment:
what is the right way to compare two objects ? 
it's better to use streams here, will make the code shorter